rust/gui-client/src-common/src/controller.rs

```diff
@@ -382,7 +382,10 @@ impl<I: GuiIntegration> Controller<'_, I> {
             .auth
             .handle_response(auth_response)
             .context("Couldn't handle auth response")?;
+
+        self.update_telemetry_context().await?;
```

Comment:
If this fails or hangs for some reason (Sentry is down), would that block the client from signing in?

Reply:
No, see the definition of this function: The only failure point here is if we can't send a message to our IPC service.